### DISCUSSION TITLE
no way to switch to cycle page mode on mac

### DISCUSSION THREAD

**Discussion by George-w1234** [bitfocus/companion#2447] [General]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

is there meant to be a non infinite mode on the web buttons for the mac version. i cant seem to find a way to get the Up down pages working, it worked a bit with an an emulator but i lost the page i was meant to be on and now cant get back to it (even though i set it to open on page 1(it still doesnt))

### Expected Behavior

there being an option to change the page mode from infinate to cycle

### Environment (please complete the following information)

```markdown
- OS: Mac Mojave
- Browser: Chrome
- Companion Version: idk fairly new
```

**Comment by Julusian** [comment]
In 3.0 the cycle mode that used to exist in tablet3 (one of the web buttons views) has been absorbed into the emulator. 
This was done as it is now possible to have multiple isolated emulators, and only the emulator is capable of respecting the full set of page navigation options.

> even though i set it to open on page 1

If you are changing the startup page in the surface settings that will only take effect when companion starts
